examples/webgl_loader_obj2_meshspray.html

```diff
@@ -1,521 +1,517 @@
+			}
+			#feedback {
+				color: darkorange;
+			}
+			#dat {
+				user-select: none;
+				position: absolute;
+				left: 0;
+				top: 0;
+				z-Index: 200;
+			}
+		</style>
+
+	</head>
+
+	<body>
+	<div id="glFullscreen">
+		<canvas id="example"></canvas>
+	</div>
+	<div id="dat">
+
+	</div>
+	<div id="info">
+		<a href="http://threejs.org" target="_blank">three.js</a> - Mesh Spray
+		<div id="feedback"></div>
+	</div>
+
+		<script src="js/Detector.js"></script>
+		<script src="../build/three.js"></script>
+		<script src="js/controls/TrackballControls.js"></script>
+		<script src="js/libs/dat.gui.min.js"></script>
+
+		<script src="js/loaders/LoaderSupport.js"></script>
+
+		<script>
+
+			'use strict';
+
+			var MeshSpray = (function () {
+
+				MeshSpray.prototype = Object.create( THREE.LoaderSupport.LoaderBase.prototype );
+				MeshSpray.prototype.constructor = MeshSpray;
+
+				function MeshSpray( manager, logger ) {
+					THREE.LoaderSupport.LoaderBase.call( this, manager, logger );
+					this.workerSupport = null;
+					this.logger = new THREE.LoaderSupport.ConsoleLogger();
+				};
+
+				MeshSpray.prototype.run = function ( prepData, workerSupportExternal ) {
+
+					if ( THREE.LoaderSupport.Validator.isValid( workerSupportExternal ) ) {
+
+						this.workerSupport = workerSupportExternal;
+						this.logger = workerSupportExternal.logger;
+
+					} else {
+
+						this.workerSupport = THREE.LoaderSupport.Validator.verifyInput( this.workerSupport, new THREE.LoaderSupport.WorkerSupport() );
+
+					}
+
+					this.logger.logTimeStart( 'MeshSpray' );
+
+					this._applyPrepData( prepData );
+
+					var scope = this;
+					var scopeBuilderFunc = function ( payload ) {
+						var meshes = scope.builder.processPayload( payload );
+						var mesh;
+						for ( var i in meshes ) {
+							mesh = meshes[ i ];
+							scope.loaderRootNode.add( mesh );
+						}
+					};
+					var scopeFuncComplete = function ( message ) {
+						var callback = scope.callbacks.onLoad;
+						if ( THREE.LoaderSupport.Validator.isValid( callback ) ) callback(
+							{
+								detail: {
+									loaderRootNode: scope.loaderRootNode,
+									modelName: scope.modelName,
+									instanceNo: scope.instanceNo
+								}
+							}
+						);
+						scope.logger.logTimeEnd( 'MeshSpray' );
+					};
+
+					var buildCode = function ( funcBuildObject, funcBuildSingelton ) {
+						var workerCode = '';
+						workerCode += '/**\n';
+						workerCode += '  * This code was constructed by MeshSpray buildCode.\n';
+						workerCode += '  */\n\n';
+						workerCode += 'THREE.LoaderSupport = {};\n\n';
+						workerCode += funcBuildObject( 'THREE.LoaderSupport.Validator', THREE.LoaderSupport.Validator );
+						workerCode += funcBuildSingelton( 'THREE.LoaderSupport.ConsoleLogger', THREE.LoaderSupport.ConsoleLogger );
+						workerCode += funcBuildSingelton( 'Parser', Parser );
+
+						return workerCode;
+					};
+					var libs2Load = [ 'node_modules/three/build/three.js' ];
```

Comment:
`node_modules/three/build/three.js`?

Reply:
It should be `build/three.min.js`. Thanks.
There are other path issues in other examples. I will double check now and push an update.